spikeinterface/core/baserecording.py

```diff
@@ -564,8 +564,11 @@ def __init__(self, sampling_frequency=None, t_start=None, time_vector=None):
 
     def get_times(self):
         if self.time_vector is not None:
+            import h5py
             if isinstance(self.time_vector, np.ndarray):
                 return self.time_vector
+            elif isinstance(self.time_vector, h5py.Dataset):
+                return self.time_vector[:]
             else:
                 return self.time_vector.asarray()
```

Comment:
I would propose here  `return np.array( self.time_vector)` so no elif and no dep to h5py.
Does it works ?